framework/utils/xmlUtils.py

```diff
@@ -206,16 +206,16 @@ def fixXmlText(msg):
     @ Out, msg, string, fixed string
   """
   #if not a string, pass it back through
-  if not isinstance(msg,basestring):
+  if not type(msg).__name__ in ['unicode','str']:
```

Comment:
@joshua-cogliati-inl, it looks like the problem is how `WriteTex.py` was importing the module. I propose two changes to ensure this works:

1. Add the following line to `framework/utils/__init.py__`
```python
from __future__ import absolute_import
```
2. Change the following line in `doc/user_guide/writeTex.py`:
```python
- sys.path.append(os.path.join(os.getcwd(),'..','..','framework','utils'))
- import xmlUtils
+ sys.path.append(os.path.join(os.getcwd(),'..','..','framework'))
+ from utils import xmlUtils
```